src/Components/Onboarding/Hooks/useOnboardingContext.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-relative-import-paths/no-relative-import-paths */
```

Comment:
Why add this?

Reply:
Yeah just autofix the errors

Reply:
I didn't... I thought you did @dzucconi. 😆 I'll remove.

Reply:
@damassi liveshare must be haunted. ;)

Reply:
its all madness 😅